### DISCUSSION TITLE
question: empty `nlsLabelKey`

### DISCUSSION THREAD

**Discussion by vince-fugnitto** [eclipse-theia/theia#11562] [Q&A] [answered]
@msujew I had a question regarding a command I noticed in `@theia/output`.
Is it correct to call `toLocalizedCommand` given that there is no `label` and we do not provide an `nlsLabelKey` (set it to `''`)?

https://github.com/eclipse-theia/theia/blob/fe224bf04e371f054c7fa059bfc5ba9da99b8e07/packages/output/src/browser/output-commands.ts#L54-L58

**Comment by msujew** [comment, marked as answer]
@vince-fugnitto Uhm, I believe I was in an auto-pilot mode while running through the code and applying all the i18n changes. I probably just assumed that since there's a `category` property, it'll need to be translated in some way. But I believe the `category` is superfluous for that command anyway, right?

Refactoring it into the following seems reasonable to me:

```ts
export const CLEAR__WIDGET: Command = { 
    id: 'output:widget:clear', 
    iconClass: codicon('clear-all') 
}; 
```

**Reply by vince-fugnitto** [reply]
Thanks!
